tests/tests/proposal/proposal_tests.cpp

```diff
@@ -7,6 +7,7 @@
 
 #include <scorum/chain/dbs_registration_committee.hpp>
 #include <scorum/chain/dbs_proposal.hpp>
+#include <scorum/chain/dbs_dynamic_global_property.hpp>
 
 #include "actor.hpp"
 #include "genesis.hpp"
```

Comment:
Fix tests/CMakeLists.txt to see all hpp-files in project (etc. /tests/proposal/genesis.hpp). 
Move genesis.hpp, actor.* to common. It is useful not only for 'proposal' test.